service/src/main/java/uk/gov/mca/beacons/api/accountholder/application/AccountHolderService.java

```diff
@@ -59,28 +67,41 @@ public Optional<AccountHolder> updateAccountHolder(
     AccountHolderId id,
     AccountHolder accountHolderUpdate
   ) {
-    AccountHolder accountHolder = accountHolderRepository
-      .findById(id)
-      .orElse(null);
-    if (accountHolder == null) return Optional.empty();
+    try {
+      String eviesUserId = "b96c194c-9e1c-4869-abdf-3d0e854c111d";
```

Comment:
👀

Reply:
Oops that was for testing against a user in the testmcga tenant b2c! Removed